convert_db.py

```diff
@@ -0,0 +1,88 @@
+import copy
+import os
+
+import pymongo
+from mongoengine import connect, register_connection
+from mongoengine.context_managers import switch_db
+
+from models.shreds import Shred, Cluster, ClusterMember
+
+# Does not overwrite data. Reads from 'clusters' collection, writes to 'cluster'
+# and 'shred'.
+SOURCE_DB_NAME = "unshred-staging"
```

Comment:
Ideally it should be taken from sys.argv (cause argparse will be a bit of overkill)